scripts/mkcloud

```diff
@@ -774,9 +776,57 @@ function testsetup()
     onadmin testsetup
     local ret=$?
     $scp root@$adminip:tempest.log "$artifacts_dir/tempest.log"
+
+    # Register the cloud in Rally and import the results
+    if [ -n "$rally_server" ]; then
```

Comment:
nitpicking: this could also be `[[ $rally_server ]]` instead of `[ -n "$rally_server" ]`

FYI: Sooner or later I will create a cleanup PR to make this consistent in mkcloud. Using `[[` now will prevent this code line from being touched again.